docs/markdown/snippets/pkgconfig-requires.md

```diff
@@ -0,0 +1,4 @@
+## pkgconfig.generate supports internal dependencies in `requires`
```

Comment:
This should be added directly to docs/markdown/Release-notes-for-1.9.0.md as snippets will end up in the 1.10 release notes instead.